### PR TITLE
Skip sorting inventory after trade

### PR DESCRIPTION
**Pull request by IceKwibby**
https://github.com/Nicklason/tf2-automatic/issues/280

### CODE REVIEW

File: src/app/handler/trades.js [hunk 1]
```diff
@@ -870,8 +870,10 @@ exports.offerChanged = function (offer, oldState) {
         require('./crafting').keepMetalSupply();
 
         // Sort inventory
-        require('../crafting').sortInventory(3);
-
+        if (process.env.SKIP_SORTING_INVENTORY === 'false'){
```
Comment (Nicklason) [inline src/app/handler/trades.js]: The bot should sort the inventory by default. So it should be:

```js
process.env.SKIP_SORTING_INVENTORY !== 'false'
```

Comment (ZeusJunior) [reply src/app/handler/trades.js]: You mean !== true

Comment (aethezxd) [reply src/app/handler/trades.js]: `!== 'true'`
Shouldn't it be 'true' since it's 'false'?

Comment (ZeusJunior) [reply src/app/handler/trades.js]: Yes

Comment (Nicklason) [reply src/app/handler/trades.js]: Yeah it should have been `!== 'true'` my bad

**Comment by Nicklason** [comment]
@ZeusJunior you can just merge it when it has been approved

**Pull request merged**